src/styles/utilities/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$onHoverCardTextColor: #34568b;
+$onHoverNavTextColor: #00acdc;
```

Comment:
This colours can be used for other purposes too, so better to name them as it looks like (grey, blue, etc.) or as what they used for (text-main, text-secondary, etc.)